src/structure/BoundingVolume.ts

```diff
@@ -0,0 +1,5 @@
+export interface BoundingVolume {
+  region?: number[];
+  box?: number[];
+  sphere?: number[];
```

Comment:
Feature request: S2 support.

Otherwise certain tilesets don't validate (as expected)

```
    {
      "type": "ANY_OF_ERROR",
      "path": "/root/children/1/(level 0, (0,0))/(level 1, (1,0))/(level 2, (2,1))/(level 3, (5,2))/(level 4, (11,4))/boundingVolume",
      "message": "The 'boundingVolume' must define 'box', 'region', or 'sphere', but not does not define any of them",
      "severity": "ERROR"
    },
```

Reply:
This is largely covered by the IMPLEMENTATION.md -> Discussion Points -> JSON Schema Based Validation. 

The point there is to be able to add ~"something like" the S2 validation, based on a plain schema. 
The fact that this will affect the behavior of the existing validation is the main thing that makes this non-trivial (but I have some ideas about how that _might_ be solved). 

(This is, in fact, a case that is ~"somewhat structurally similar" to the question of how extensions may, for example, define additional valid `enum` values. Not entirely trivial, but certainly solvable)